packages/fxa-content-server/tests/functional/send_sms.js

```diff
@@ -202,56 +203,28 @@ const suite = {
     'empty phone number': function() {
       return this.remote
         .then(openPage(SEND_SMS_URL, selectors.SMS_SEND.HEADER))
-        .then(click(selectors.SMS_SEND.SUBMIT))
-        .then(testElementExists(selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP))
-        .then(
-          testElementTextInclude(
-            selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP,
-            'required'
-          )
-        );
+        .then(testElementDisabled(selectors.SMS_SEND.SUBMIT));
     },
 
     'invalid phone number (too short)': function() {
       return this.remote
         .then(openPage(SEND_SMS_URL, selectors.SMS_SEND.HEADER))
         .then(type(selectors.SMS_SEND.PHONE_NUMBER, '2134567'))
-        .then(click(selectors.SMS_SEND.SUBMIT))
-        .then(testElementExists(selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP))
-        .then(
-          testElementTextInclude(
-            selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP,
-            'invalid'
-          )
-        );
+        .then(testElementDisabled(selectors.SMS_SEND.SUBMIT));
     },
 
     'invalid phone number (too long)': function() {
       return this.remote
         .then(openPage(SEND_SMS_URL, selectors.SMS_SEND.HEADER))
         .then(type(selectors.SMS_SEND.PHONE_NUMBER, '21345678901'))
-        .then(click(selectors.SMS_SEND.SUBMIT))
-        .then(testElementExists(selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP))
-        .then(
-          testElementTextInclude(
-            selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP,
-            'invalid'
-          )
-        );
+        .then(testElementDisabled(selectors.SMS_SEND.SUBMIT));
     },
 
     'invalid phone number (contains letters)': function() {
       return this.remote
         .then(openPage(SEND_SMS_URL, selectors.SMS_SEND.HEADER))
         .then(type(selectors.SMS_SEND.PHONE_NUMBER, '2134567a890'))
-        .then(click(selectors.SMS_SEND.SUBMIT))
-        .then(testElementExists(selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP))
-        .then(
-          testElementTextInclude(
-            selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP,
-            'invalid'
-          )
-        );
```

Comment:
I think we want to leave at least one of these tests in here, there's additional validations server-side:

![image](https://user-images.githubusercontent.com/13018240/76114813-8cd4ee80-5fac-11ea-863d-ab97f6de2d6e.png)

Reply:
Good call. I've added an override to disable input-based validation so we can test both bad and no input.